lib/matplotlib/colors.pyi

```diff
@@ -409,6 +414,44 @@ class BoundaryNorm(Normalize):
 
 class NoNorm(Normalize): ...
 
+class MultiNorm(Norm):
+    # Here "type: ignore[override]" is used for functions with a return type
+    # that differs from the function in the base class.
+    # i.e. where `MultiNorm` returns a tuple and Normalize returns a `float` etc.
+    def __init__(
+        self,
+        norms: ArrayLike,
+        vmin: ArrayLike | float | None = ...,
+        vmax: ArrayLike | float | None = ...,
+        clip: ArrayLike | bool = ...
+    ) -> None: ...
+    @property
+    def norms(self) -> tuple[Normalize, ...]: ...
+    @property  # type: ignore[override]
+    def vmin(self) -> tuple[float | None, ...]: ...
+    @vmin.setter
+    def vmin(self, values: ArrayLike | float | None) -> None: ...
+    @property  # type: ignore[override]
+    def vmax(self) -> tuple[float | None, ...]: ...
+    @vmax.setter
+    def vmax(self, valued: ArrayLike | float | None) -> None: ...
+    @property  # type: ignore[override]
+    def clip(self) -> tuple[bool, ...]: ...
+    @clip.setter
+    def clip(self, values: ArrayLike | bool) -> None: ...
+    @overload
+    def __call__(self, values: tuple[np.ndarray], clip: ArrayLike | bool | None = ...) -> tuple[np.ndarray]: ...
+    @overload
+    def __call__(self, values: tuple[float], clip: ArrayLike | bool | None = ...) -> tuple[float]: ...
```

Comment:
Is this really supposed to be a tuple of a single float? Or a single float (`float`)? Or a tuple of floats (`tuple[float, ...]`)?